mutis/correlation.py

```diff
@@ -75,14 +75,21 @@ def gen_synth(self, samples):
         self.signal1.gen_synth(samples)
         self.signal2.gen_synth(samples)
 
-    def gen_corr(self):
+    def gen_corr(self, uncert=True, dsamples=500):
         """Generates the correlation of the signals.
 
         Generates the correlation of the signals, and computes their
         confidence level from the synthetic light curves, which must
         have been generated before.
         """
 
+        if (uncert is True) and (self.signal1.dvalues is None):
```

Comment:
- Simple is better than complex
-  Sparse is better than dense.
`if uncert and self.signal1.dvalues is None:`

Reply:
Ok, I find the `if (uncert is True) and (self.signal1.dvalues is None)`{:.python} less confusing since sometimes -logical- operator precedence is tricky but I guess that is the pythonic way...